feat: Add USD viewer integration (#486)
ansys/ansys-tools-visualization-interface: doc/source/user_guide/usdviewer.rst

.. _ref_usd_viewer:

USD viewer usage
################

The Ansys Tools Visualization Interface provides a USD viewer that can be

> used to visualize USD stages. The viewer is built on top of the python-usd-viewer package,

Reviewer: We should use quotes with package names
```suggestion
used to visualize USD stages. The viewer is built on top of the ``python-usd-viewer`` package,
```

which provides a simple interface for visualizing USD stages.

USD viewer installation
-----------------------

> To use the USD viewer, you need to install the python-usd-viewer package. Please refer

Reviewer: ```suggestion
To use the USD viewer, you need to install the ``python-usd-viewer`` package. Please refer
```

to the `USD viewer installation guide`_ for detailed instructions on how to install the package.

> .. _USD viewer installation guide: https://github.com/ansys/python-usd-viewer

Reviewer: This page you refer to doesn't include info on how to install the package from PyPI - we should add it

Author: I updated to point to the getting started guide for it and opened an issue for it in https://github.com/ansys/python-usd-viewer/issues/69

> .. _USD viewer installation guide: https://github.com/ansys/python-usd-viewer

Author: ```suggestion
.. _USD viewer installation guide: https://usd-viewer.docs.pyansys.com/version/stable/getting_started/index.html
```

> ..

Reviewer: Any reason for this rST comment?

Author: This is a way to add external links in RST: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

Changed it to inline link, it aligns better with the rest of our docs.

USD viewer usage
----------------

Once you have the USD viewer installed, you can use it to visualize USD stages.
The Ansys Tools Visualization Interface provides a `USDInterface` class that serves
as a bridge between the USD viewer and the Ansys Tools Visualization Interface plotter.

> Here's a simple example of how to use the USD viewer through the Ansys Tools Visualization Interface:

Reviewer: ```suggestion
Here is a simple example of how to use the USD viewer through the Ansys Tools Visualization Interface:
```
Avoid contractions
